aip/general/0193.md

```diff
@@ -138,6 +138,18 @@ descriptions, and examples:
     `ErrorInfo`,`LocalizedMessage`, and so on.
   - *Example*: Described in the [ErrorDetails](#error-details) section.
 
+
+**Important:** In the context of the of the `google.rpc.Status` Protobuf
+message, the value of the field `code` is the numeric equivalent of the
+enum value chosen from [`google.rpc.Code`][Code]. For example, if
+[`INVALID_ARGUMENT`][InvalidArgument] is chosen, the value of `code`
+will be `3` (in the context of the Protobuf message,
+`google.rpc.Status`). However, when the error is expressed as JSON, as
+in the [sample above](#json-representation), the value of the field by
+the same name, `"code"`, is the HTTP status code equivalent of the
```

Comment:
Personally I'd say "equivalent to" rather than "equivalent of" but that's a nit-pick.

Reply:
Although oddly I don't have a problem earlier with "numeric equivalent" - I think in this case it's being used as a verb (with HTTP status code being the noun) whereas earlier equivalent *is* the noun (with numeric as an adjective).

Reply:
Changed to "equivalent to".